src/main/java/com/bookstore/controller/OrderItemController.java

```diff
@@ -0,0 +1,34 @@
+package com.bookstore.controller;
+
+import com.bookstore.dto.order.OrderItemDto;
+import com.bookstore.service.order.OrderItemService;
+import io.swagger.v3.oas.annotations.tags.Tag;
+import java.util.List;
+import lombok.RequiredArgsConstructor;
+import org.springframework.security.access.prepost.PreAuthorize;
+import org.springframework.web.bind.annotation.GetMapping;
+import org.springframework.web.bind.annotation.PathVariable;
+import org.springframework.web.bind.annotation.RequestMapping;
+import org.springframework.web.bind.annotation.RestController;
+
+@Tag(name = "OrderItem", description = "Order Item API")
+@RestController
+@RequestMapping("/orders/{orderId}/items")
+@RequiredArgsConstructor
+public class OrderItemController {
```

Comment:
all these logic should be in OrderController, so remove OrderItemController and replace endpoints